nifi-frontend/src/main/frontend/apps/nifi/src/app/app.module.ts

```diff
@@ -96,8 +92,14 @@ import { BannerTextEffects } from './state/banner-text/banner-text.effects';
     ],
     providers: [
         { provide: MAT_FORM_FIELD_DEFAULT_OPTIONS, useValue: { appearance: 'outline' } },
-        provideHttpClient(withInterceptors([authInterceptor, loadingInterceptor, pollingInterceptor]))
-    ],
-    bootstrap: [AppComponent]
+        provideHttpClient(withInterceptors([authInterceptor, loadingInterceptor, pollingInterceptor])),
+        provideHttpClient(
+            withInterceptorsFromDi(),
```

Comment:
I think we can be condensed. All of our interceptors have been converted to function based so I don't think we need the `widthInterceptorsFromDi`.